pep-0723.rst

```diff
@@ -270,9 +301,13 @@ about metadata itself direct users to the living document that describes
 `project metadata <pyproject metadata_>`_.
 
 We will document that the name and version fields in the ``[project]`` table
-may be elided for simplicity. Additionally, we will have guidance (perhaps
-temporary) explaining that single-file scripts cannot be built into a wheel
-and therefore you would never see the associated ``[build-system]`` metadata.
+may be elided for simplicity. Additionally, we will have guidance explaining
+that single-file scripts cannot (yet) be built into a wheel via standard means.
+
+We will explain that it is up to individual tools whether or not fields are
+read and alter behavior. For example, every script runner may not be able to
```

Comment:
Grammar doesn't make sense here ('are read and alter')

Reply:
I reworded this